src/test/java/org/opentripplanner/middleware/ApiKeyManagementTest.java

```diff
@@ -61,13 +61,10 @@ public static void setUp() throws IOException, InterruptedException {
      */
     @AfterAll
     public static void tearDown() {
-        assumeTrue(isEndToEnd);
         setAuthDisabled(getDefaultAuthDisabled());
+        apiUser.delete(false);
```

Comment:
This line is causing a CI test failure: https://travis-ci.org/github/ibi-group/otp-middleware/builds/729933382#L389

Reply:
@binh-dam-ibigroup ... but works locally (I know!). I can't see how env variables are impacting this, but I can't think of anything else. Any ideas?

Reply:
@binh-dam-ibigroup I am actually get an error message but I don't think it is related. "Must be an admin to delete an API key", I'll need to update to delete the user as an admin.

Reply:
@binh-dam-ibigroup Ignore that it is expected as part of the delete tests. I'm having a good day!